blob/rollup.config.js

```diff
@@ -0,0 +1,61 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import { uglify } from 'rollup-plugin-uglify';
+import replace from 'rollup-plugin-replace';
+import commonjs from 'rollup-plugin-commonjs';
+import shim from 'rollup-plugin-shim';
+
+export default [
+  {
+    external: ['ms-rest-js', 'crypto', 'fs', 'events', 'os'],
+    input: 'dist-esm/lib/index.js',
+    output: {
+      file: 'dist/index.js',
+      format: 'cjs',
+      sourcemap: true
+    },
+    plugins: [nodeResolve({ module: true }), uglify()]
+  },
+  {
+    external: ['ms-rest-js'],
```

Comment:
Before ms-rest-js provides backward compatibility. Will take in this PR but will keep ms-rest-js included in the bundle.

Wait @RikkiGibson's https://github.com/Azure/ms-rest-js/issues/231 's fix before merging.